Packs/Troubleshoot/Scripts/TroubleshootExecutePlaybookByAlertQuery/TroubleshootExecutePlaybookByAlertQuery.py

```diff
@@ -166,22 +178,31 @@ def handle_results(command_results: dict, playbook_id: str, alert_ids: list, res
         return None
 
     try:
-        if "The request requires the right permissions" in command_results[0].get('Contents'):
-            return_error("Request Failed: Insufficient permissions. Ensure the API key has the appropriate access rights.")
-
-        result_dict = command_results[0].get('Contents', {}).get('response', {})
+        if isinstance(alert_ids, str):
+            result_dict = command_results[0].get('Contents', {})
+            if type(result_dict) == str:
+                results_summary.update_failure_create(playbook_id, alert_ids)
+                return None
 
-        if not result_dict:
             results_summary.update_success(playbook_id, alert_ids)
-            return None
 
-        failed_ids = list(result_dict.keys())
-        succeeded_ids = list(set(alert_ids) - set(failed_ids))
+        elif isinstance(alert_ids, list):
+            if "The request requires the right permissions" in command_results[0].get('Contents'):
+                return_error("Request Failed: Insufficient permissions. Ensure the API key has the appropriate access rights.")
+
+            result_dict = command_results[0].get('Contents', {}).get('response', {})
 
-        results_summary.update_failure_create(playbook_id, failed_ids)
+            if not result_dict:
+                results_summary.update_success(playbook_id, alert_ids)
+                return None
```

Comment:
You can remove the none 
```suggestion
                return
```